client/landing/stepper/declarative-flow/internals/steps-repository/videopress-onboarding-intent/styles.scss

```
@@ -1,61 +1,310 @@
-.videopress-onboarding-intent {
-	max-width: 1200px !important;
+.videopress.intro .signup-header {
+	z-index: 0;
 }
 
-.videopress-onboarding-intent__step-content {
-	display: grid;
-	grid-template: "a b c" 1fr;
-	grid-row-gap: 48px;
-	justify-items: center;
-}
+.videopress-onboarding-intent {
+	max-width: 1200px !important;
+	font-family: "SF Pro Display", $sans;
 
-@media (max-width: 1060px) {
 	.videopress-onboarding-intent__step-content {
-		grid-template: "a b" 1fr;
+		display: grid;
+		grid-template: "a b c" 1fr;
+		grid-row-gap: 48px;
+		justify-items: center;
 	}
-}
 
-@media (max-width: 760px) {
-	.videopress-onboarding-intent__step-content {
-		grid-template: "a" 1fr;
+	@media (max-width: 1060px) {
+		.videopress-onboarding-intent__step-content {
+			grid-template: "a b" 1fr;
+		}
 	}
-}
-
-.videopress-intent-item__preview {
-	width: 328px;
-	height: 220px;
-	overflow: hidden;
-	border: solid 8px #212121;
-	border-radius: 8px; /* stylelint-disable-line */
-	cursor: pointer;
-	transition: border-color 0.3s ease-in-out;
 
-	&:hover {
-		border-color: #ffe61c;
+	@media (max-width: 760px) {
+		.videopress-onboarding-intent__step-content {
+			grid-template: "a" 1fr;
+		}
 	}
-}
 
-.videopress-intent-item {
-	display: flex;
-	flex-direction: column;
-	gap: 12px;
-	width: 328px;
-}
+	.videopress-intent-item__preview {
+		width: 328px;
+		height: 220px;
+		overflow: hidden;
+		border: solid 8px #212121;
+		border-radius: 8px; /* stylelint-disable-line */
+		cursor: pointer;
+		transition: border-color 0.3s ease-in-out;
 
-.videopress-intent-item__description {
-	display: flex;
-	flex-direction: column;
-	text-align: center;
-	gap: 4px;
+		&:hover {
+			border-color: #ffe61c;
+		}
+	}
 
-	.videopress-intent-item__title {
-		font-size: 17px; /* stylelint-disable-line */
-		font-weight: 600;
+	.videopress-intent-item {
+		display: flex;
+		flex-direction: column;
+		gap: 12px;
+		width: 328px;
 	}
 
 	.videopress-intent-item__description {
-		font-size: 14px; /* stylelint-disable-line */
+		display: flex;
+		flex-direction: column;
+		text-align: center;
+		gap: 4px;
+
+		.videopress-intent-item__title {
+			font-size: 17px; /* stylelint-disable-line */
+			font-weight: 600;
+		}
+
+		.videopress-intent-item__description {
+			font-size: 14px; /* stylelint-disable-line */
+			font-weight: 400;
+			color: #a7aaad;
+		}
+	}
+
+	.intro__more-modal {
+		position: absolute;
```

```suggestion
		position: fixed;
```

Set to `fixed` so it fills the viewport instead of being stuck to the page. Fixes an issue when you're scrolled half-way down the page in responsive mode.